docs: fill in gaps from v15-16 (#658)
juju/terraform-provider-juju: docs-rtd/howto/manage-groups.md

(manage-groups)=
# Manage groups

```{note}
In the Juju ecosystem, groups are supported only when using [JAAS](https://canonical-jaas-documentation.readthedocs-hosted.com/en/latest/).
```

## Reference an externally managed group

To reference a group you've created with Juju tools other than the Terraform Provider for Juju, in your Terraform plan add a data source of the `juju_jaas_group` type, specifying the name of the group. For example:

```terraform
data "juju_jaas_group" "test" {
  name = "group-0"
}
```

> See more: [`juju_jaas_group` (data source)](https://registry.terraform.io/providers/juju/juju/latest/docs/data-sources/jaas_group)

## Add a group

> To add a group, in your Terraform plan create a resource of the `juju_jaas_group` type, specifying, at the very least, a name. For example:

Reviewer: ```suggestion
To add a group, in your Terraform plan create a resource of the `juju_jaas_group` type, specifying its name. For example:
```

```terraform
resource "juju_jaas_group" "development" {
  name = "devops-team"
}
```

> See more: [`juju_jaas_group` (resource)](https://registry.terraform.io/providers/juju/juju/latest/docs/resources/jaas_group)

(manage-access-to-a-group)=
## Manage access to a group

> When using Juju with JAAS, to grant one or more users, groups, and/or service accounts access to a group, in your Terraform plan add a resource type `juju_jaas_access_group`, specifying the group ID, the JAAS group access level, and the list of desired users, groups, and/or service accounts. For example:

Reviewer: Thinking that maybe we should reword it to make it cleared that by "adding access to a group" we manage group membership.. "granting" somebody access to a group effectively makes them a member of that group.
